src/lem/pointers.rs

```diff
@@ -46,64 +32,14 @@ impl<F: LurkField> Ptr<F> {
         }
     }
 
-    #[inline]
-    pub fn num(f: F) -> Self {
-        Ptr::Atom(Tag::Expr(Num), f)
-    }
-
-    #[inline]
-    pub fn num_u64(u: u64) -> Self {
-        Ptr::Atom(Tag::Expr(Num), F::from_u64(u))
-    }
-
-    #[inline]
-    pub fn u64(u: u64) -> Self {
-        Ptr::Atom(Tag::Expr(U64), F::from_u64(u))
-    }
-
-    #[inline]
-    pub fn char(c: char) -> Self {
-        Ptr::Atom(Tag::Expr(Char), F::from_char(c))
-    }
-
-    #[inline]
-    pub fn comm(hash: F) -> Self {
-        Ptr::Atom(Tag::Expr(Comm), hash)
-    }
-
-    #[inline]
-    pub fn zero(tag: Tag) -> Self {
-        Ptr::Atom(tag, F::ZERO)
-    }
-
-    pub fn is_zero(&self) -> bool {
-        match self {
-            Ptr::Atom(_, f) => f == &F::ZERO,
-            _ => false,
-        }
-    }
-
     pub fn is_nil(&self) -> bool {
         self.tag() == &Tag::Expr(Nil)
     }
 
-    #[inline]
-    pub fn dummy() -> Self {
-        Self::zero(Tag::Expr(Nil))
-    }
-
-    /// Creates an atom pointer from a `ZPtr`, with its tag and hash. Thus hashing
-    /// such pointer will result on the same original `ZPtr`
-    #[inline]
-    pub fn opaque(z_ptr: ZPtr<F>) -> Self {
-        let crate::z_data::z_ptr::ZPtr(t, h) = z_ptr;
-        Ptr::Atom(t, h)
-    }
```

Comment:
Nit: we may want to keep this one for didactic uses.

Reply:
It's just moved to `Store` 👍🏼